### DISCUSSION TITLE
Is it possible to have a path in a transient prompt?

### DISCUSSION THREAD

**Discussion by LittleLilyBun** [JanDeDobbeleer/oh-my-posh#5697] [Troubleshoot] [answered]
I want the path I was in when running a command to appear in that command's transient prompt, but when I add `{{ .Path }}` to the transient prompt, I only get an error in the transient prompt. Is it possible to have the path show up in a transient prompt?

This is what my transient prompt looks like:
```
[transient_prompt]
  type = 'text'
  style = 'plain'
  template = '{{ .Path }} 󰅂 '
  background = 'transparent'
  foreground = '#7AB0E0'
```
and the transient prompt ends up looking like this: `: nil pointer evaluating template.Data.Path`

Sorry if I'm just missing something obvious in the documentation.

**Comment by JanDeDobbeleer** [comment, marked as answer]
You can use `{{ .PWD }}`

**Reply by LittleLilyBun** [reply]
This worked! Thank you
